pkg/cloud/azure/services/networkinterfaces/networkinterfaces.go

```diff
@@ -82,28 +84,37 @@ func (s *Service) CreateOrUpdate(ctx context.Context, spec azure.Spec) error {
 	nicHasIPv6 := subnetHasIPv6(subnet)
 
 	nicProp := network.InterfacePropertiesFormat{}
+
+	// Enaled Accelerated networking
+	if s.Scope.MachineConfig.AcceleratedNetworking {
+		if !machineset.InstanceTypes[s.Scope.MachineConfig.VMSize].AcceleratedNetworking {
+			return errors.New("accelerated networking not supported on instance type " + s.Scope.MachineConfig.VMSize)
+		}
+		klog.V(3).Infof("setting EnableAcceleratedNetworking to %v", s.Scope.MachineConfig.AcceleratedNetworking)
```

Comment:
Level three is the default level we run at, this kind of detail I'd say should be at a higher level. I'd move this to level 4